CHANGELOG-fix-minor-publication-errors.md

```diff
@@ -0,0 +1,6 @@
+- Update from deprecated createMuiTheme function to createTheme
+- Add propTypes entry for vignette_json
+- Fix proptypes error with DOI string being passed as boolean
+- Fix semantically invalid HTML nesting errors (ul inside of p, p inside of p)
+- Import Accordion instead of deprecated ExpansionPanel
+- Add handling for SUBMITTED status to StatusIcon
```

Comment:
```suggestion
- Update from deprecated createMuiTheme function to createTheme.
- Add propTypes entry for vignette_json.
- Fix proptypes error with DOI string being passed as boolean.
- Fix semantically invalid HTML nesting errors (ul inside of p, p inside of p).
- Import Accordion instead of deprecated ExpansionPanel.
- Add handling for SUBMITTED status to StatusIcon.
```

Reply:
Trivial, but I think we've been consistent about this in the past. Might have missed it in your first PR.

Reply:
We cold probably enforce this somewhere if it's worth the effort.

Reply:
I'll keep this in mind, thanks for the correction 👍